recipes/libcurl/all/conanfile.py

```diff
@@ -1,13 +1,20 @@
-import glob
-import os
-import re
-import functools
-from conans import ConanFile, AutoToolsBuildEnvironment, CMake, tools
-from conans.errors import ConanInvalidConfiguration
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.apple import is_apple_os, fix_apple_shared_install_name
 from conan.tools.microsoft import is_msvc
 from conan.tools.build import cross_building
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.files import apply_conandata_patches, copy, download, get, load, replace_in_file, rm, rmdir, save
+from conan.tools.gnu import Autotools, AutotoolsToolchain, AutotoolsDeps, PkgConfigDeps
+from conan.tools.microsoft.subsystems import unix_path
+from conan.tools.scm import Version
+# TODO: Migrate
+from conans.tools import get_env, os_info
 
-required_conan_version = ">=1.45.0"
+import os
+import re
+
+required_conan_version = ">=1.50.0"
```

Comment:
```suggestion
required_conan_version = ">=1.51.3"
```
from conan.tools.apple import is_apple_os: https://docs.conan.io/en/latest/changelog.html#aug-2022